### DISCUSSION TITLE
Daily 08/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#46] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve tratando de ponerme al día con los apuntes y ejercicios de ciberseguridad que se encuentran en la plataforma. Quería hacer, también el ejercicio de app pero no me alcanzó el tiempo. Por la tarde participé de la clase de ciberseguridad.
Hoy cuento con poco tiempo, también. Trataré de realizar alguno de los ejercicios que me quedaron pendientes de ayer.

**Comment by BraianTroncoso** [comment]
Estuve con la clase de Ingles, y seguí explorando para ver si encuentraba proyectos similares para tomar referencias.
